### ISSUE TITLE
Tracking comments for an issue : Issue Assignees, Linked Project board Info.

### ISSUE THREAD

**Issue by suppathak** [thoth-station/mi#589]
## Problem statement
Tracking comments for an issue : Issue Assignees, Linked Project board info. 
 
## High-level Goals
In order to build a community around thoth guidance service. I want to get an idea about community by visualizing metrics like number of assignees, if the issue is linked to any project board or not, etc for [thoth-station support](https://github.com/thoth-station/support) repo.

## Proposal description
I want to use srcopsmetrics in order to extract the comment level activity for an issue.

## Acceptance Criteria

Able to track metrics like:
- [x] Assignees
- [x] Linked project board info.

**Comment by xtuchyna** [comment]
Hello @suppathak , thanks for creating Issue!

PR is on the way, it could also help if you could specify ` Linked project board info.` in more detail. I suppose you want these metadata:
- project url
- column name

Is there something else that should be included? Use [project metadata](https://docs.github.com/en/rest/projects/projects#get-a-project) as a reference and please let us know! :)

**Comment by suppathak** [comment]
> * project url
> * Issue's column name inside the project board

Hey @xtuchyna , 

Yes I want these two metadata related to the project board linked to issue. Thanks :)

**Comment by xtuchyna** [comment]
Hey @suppathak , for which repositories are you going to extract the project-related information? Is it just `thoth-station` ?

**Comment by suppathak** [comment]
> Hey @suppathak , for which repositories are you going to extract the project-related information? Is it just `thoth-station` ?

Hey @xtuchyna. Yes! I am planning to extract these metrics for `thoth-station/support` repo. 
Thanks!

**Comment by xtuchyna** [comment]
Hey @suppathak , project event details cannot be still retrieved, even after enabling projects thru thoth-station/core
That means you won't be able to use the feature on thoth-station/support and also that I cannot test the feature in #591 
Please stay tuned in https://github.com/thoth-station/core/issues/419 to see updates on this matter

**Comment by xtuchyna** [comment]
So, in the end, there seems to be no way we can get this information by interacting with GH API (https://github.com/thoth-station/srcops-testing/issues/269 and other tests)

If we are still interested in this meta-information, it could be still scraped from web. WDYT @suppathak

**Comment by VannTen** [comment]
It seems to me reading the documentation (in particular, issue events [added_to_project](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#added_to_project) and [removed_to_project](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#added_to_project)) that this should be possible unless I misunderstood the issue

**Comment by xtuchyna** [comment]
> It seems to me reading the documentation (in particular, issue events [added_to_project](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#added_to_project) and [removed_to_project](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#added_to_project)) that this should be possible unless I misunderstood the issue

of course, but if you look at my comments including [this](https://github.com/thoth-station/srcops-testing/issues/265) test issue, I had no success of obtaining it (in the end custom header for these particular events did not return any of the project related events)

**Comment by xtuchyna** [comment]
@VannTen once again tried to retrieve data, but this time with new token (that had project access enabled)
but still no success

you can read all the setup details here https://github.com/thoth-station/srcops-testing/issues/271

**Comment by xtuchyna** [comment]
WDYT @goern is it still valuable to invest time in this?

**Comment by xtuchyna** [comment]
STATUS update:
We had no success in retrieving the project related timeline events by using ordinary GH REST API (including Timeline events API). At the moment the deduction is that the newly created projects are only visible within the usage of GraphQL.

Therefore there remains one item left to try:
 
- [ ] check if we can retrieve Timeline project-related-events by GraphQL using newly created projects [meta-information test project board 1-3](https://github.com/orgs/thoth-station/projects?query=is%3Aopen+meta-information+test) and [src-ops repository](https://github.com/thoth-station/srcops-testing)

**Comment by goern** [comment]
feels to me like we should push it to the backlog?
